### PR TITLE
Prep release v0.76.2

### PR DESCRIPTION
**Pull request by djaglowski**
Revs contrib modules to v0.76.2 and retracts v0.76.1

### CODE REVIEW

File: cmd/oteltestbedcol/builder-config.yaml [hunk 1]
```diff
@@ -2,50 +2,50 @@ dist:
   module: github.com/open-telemetry/opentelemetry-collector-contrib/cmd/oteltestbedcol
   name: oteltestbedcol
   description: OpenTelemetry Collector binary for testbed only tests.
-  version: 0.76.1-dev
+  version: 0.76.2-dev
   otelcol_version: 0.76.1
```
Comment (TylerHelmuth) [inline cmd/oteltestbedcol/builder-config.yaml:6]: Does this need bumped?

Comment (djaglowski) [reply cmd/oteltestbedcol/builder-config.yaml:6]: I believe that refers to the core collector version, which I did not need to bump.

File: exporter/datadogexporter/config.go [hunk 2]
```diff
@@ -117,7 +117,7 @@ type HistogramConfig struct {
 
 	// SendCountSum states if the export should send .sum and .count metrics for histograms.
 	// The default is false.
-	// Deprecated: [v0.75.0] Use `send_aggregation_metrics` (HistogramConfig.SendAggregations) instead.
+	// Deprecated: [v0.76.2] Use `send_aggregation_metrics` (HistogramConfig.SendAggregations) instead.
```
Comment (codeboten) [inline exporter/datadogexporter/config.go:120]: This shouldnt be changed

File: exporter/datadogexporter/examples/collector.yaml [hunk 3]
```diff
@@ -179,7 +179,7 @@ exporters:
         #
         # mode: distributions
 
-        ## Deprecated [v0.75.0]: use `send_aggreggations` instead.
+        ## Deprecated [v0.76.2]: use `send_aggreggations` instead.
```
Comment (codeboten) [inline exporter/datadogexporter/examples/collector.yaml:182]: Same here

File: exporter/datadogexporter/config.go (hunk 2 above)
Comment (djaglowski) [reply exporter/datadogexporter/config.go]: Fixed the two you mentioned and another.

Comment (codeboten) [reply exporter/datadogexporter/config.go]: Its possible github is losing its mind with this large of a changelist, but im still seeing 76.2 here

**Review by codeboten — APPROVED** [review]

**Review by TylerHelmuth — APPROVED** [review]

**Review by atoulme — APPROVED** [review]

**Review by evan-bradley — APPROVED** [review]

**Pull request merged**